modules/farm_fd2_examples/src/entrypoints/soil_disturbance/App.vue

```diff
@@ -0,0 +1,270 @@
+  <h3>Soil Disturbance Example</h3>
+  <p data-cy="description">
+    The SoilDisturbance component collects data on soil disturbances.
+  </p>
+
+  <hr />
+  <SoilDisturbance
+    id="soil-disturbance"
+    data-cy="soil-disturbance"
+    v-bind:required="required"
+    v-model:area="form.area"
+    v-model:depth="form.depth"
+    v-model:equipment="form.equipment"
+    v-model:includePasses="form.includePasses"
+    v-model:speed="form.speed"
+    v-model:passes="form.passes"
+    v-bind:showValidityStyling="validity.showStyling"
+    v-on:valid="
+      (valid) => {
+        validity.isValid = valid;
+      }
+    "
+    v-on:ready="createdCount++"
+  />
+  <hr />
+
+  <h5>Component Props:</h5>
+  <table class="example-table">
+    <thead>
+      <th>Prop</th>
+      <th>Control</th>
+    </thead>
+    <tbody>
+      <tr>
+        <td>required</td>
+        <td>
+          <BFormCheckbox
+            id="required-checkbox"
+            data-cy="required-checkbox"
+            switch
+            v-model="required"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>showValidityStyling</td>
+        <td>
+          <BFormCheckbox
+            id="styling-checkbox"
+            data-cy="styling-checkbox"
+            switch
+            v-model="validity.showStyling"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>area</td>
+        <td>
+          <BButton
+            id="set-area-button"
+            data-cy="set-area-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changeArea"
+          >
+            Area = 50
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>depth</td>
+        <td>
+          <BButton
+            id="set-depth-button"
+            data-cy="set-depth-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changeDepth"
+          >
+            Depth = 6
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>speed</td>
+        <td>
+          <BButton
+            id="set-speed-button"
+            data-cy="set-speed-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changeSpeed"
+          >
+            Speed = 3
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>Set Passes</td>
+        <td>
+          <BButton
+            id="set-passes-button"
+            data-cy="set-passes-button"
+            variant="outline-primary"
+            size="sm"
+            v-on:click="changePasses"
+          >
+            Passes = 4
+          </BButton>
+        </td>
+      </tr>
+      <tr>
+        <td>includePasses</td>
+        <td>
+          <BFormCheckbox
+            id="include-passes-checkbox"
+            data-cy="include-passes-checkbox"
+            switch
+            v-model="form.includePasses"
+          />
+        </td>
+      </tr>
+      <tr>
+        <td>Add Planter</td>
```

Comment:
Move equipment controls above the area controls and add `v-if` as appropriate to the other controls (e.g. speed, depth, etc).